libnavigator/api/current.txt

```diff
@@ -0,0 +1,14 @@
+// Signature format: 3.0
+package com.mapbox.navigation.navigator {
+
+  public final class LocationExKt {
+  }
+
+  public final class NavigationOkHttpServiceKt {
+  }
+
+  public final class NullUtilsKt {
```

Comment:
Could we avoid tracking this empty `public` class? 🤔 There are only top-level functions

Reply:
Think we should remove this right? Doesn't this mean anyone who uses the SDK will inherit our internal helper extensions

That could be a little annoying imo, everyone defines their own helper extensions

Reply:
This is the Kotlin compiler, as you can see it creates a `NullUtilsKt` class but doesn't have any `public` methods - that's why I pointed out that we need to investigate. In any case, it doesn't have any implications to clients because as mentioned there is nothing exposed.